### DISCUSSION TITLE
[windows 11 user] "Explorer Patcher" Which one is correct?

### DISCUSSION THREAD

**Discussion by JRAINKIM** [community/community#35640] [Other Feature Feedback, Questions, & Ideas]
i wondered

![Which one is correct](https://user-images.githubusercontent.com/115444389/194840006-e2862034-48b6-45f6-a5aa-799f64fcfc80.jpg)

**Comment by samus-aran** [comment]
Hi @JRAINKIM ,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on [ExplorerPatcher](https://github.com/valinet/ExplorerPatcher/) itself. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck! 🍀
